core/scss/utilities/mixins/typography/_displays.scss

```diff
@@ -0,0 +1,21 @@
+@charset "UTF-8";
+
+//
+// @displays
+//
+// Default displays shared styling.
+//
+// Style guide: Mixins.Typography.displays
+//
+@mixin displays {
+  @include font-smoothing;
+  @include margin(0 null null);
+
+  clear: both;
+  font-weight: $font-bold;
```

Comment:
```suggestion
  font-weight: $font-x-bold;
```

Reply:
Hmm....apparently in Decanter, $font-x-bold = font-weight: 700 which is "bold" in Figma, and $font-bold = font-weight: 600 is semibold in Figma. I guess all the display font in Figma actually are bold (700) so x-bold is what we need. I wonder if we should rename the variables to just be 600 = $font-semibold and $700 = $font-bold because I can see that could be a bit confusing for devs who are not us.

Reply:
I realize that my above comment might be confusing 😂 , but you could get a preview of that in the card component. I used $font-x-bold for the headline and I believe that's closer to what we want for headers.

Reply:
Haha! It makes total sense. I will clean up those variables and have them mirror what they are called in Figma as well as in the Google Fonts documentation. Good catch!

Reply:
No longer needed since variables were updated accordingly.